openapi-cli/src/test/java/io/ballerina/openapi/generators/schema/ArrayDataTypeTests.java

```diff
@@ -153,28 +154,8 @@ public void arrayHasOneOfItemsWithNullable() throws IOException, BallerinaOpenAp
                 "schema/ballerina/array_with_oneOf_complex.bal", syntaxTree);
     }
 
-    @Test(description = "Array schema has max items count that ballerina doesn't support",
```

Comment:
Shouldn't we add a negative test case and check for the warning?

Reply:
yeah, I will add test for negative tests